gflownet/envs/cube.py

```diff
@@ -249,8 +248,33 @@ def step(
         """
         pass
 
+    def _beta_params_to_policy_outputs(self, param: str, params_dict: dict):
+        """
+        Maps the values of alpha and beta given in the configuration to new values such
+        that when passed to _make_increments_distribution, the actual alpha and beta
+        passed to the Beta distribution(s) are the ones from the configuration.
+
+        Args
+        ----
+        param : str
```

Comment:
Maybe `param_name` would be a little bit more verbose?

Reply:
https://github.com/alexhernandezgarcia/gflownet/pull/214/commits/6156602bec379badbcd446de9ac08a1ba34dd379